src/amp-user-notification_with_server_endpoint.html

```diff
@@ -0,0 +1,59 @@
+<!--
+  #### Introduction
+
+  Use amp-user-notification to display a dismissable notification to the user.
+  Use this to implement a dialog to notify users about cookies.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <!-- #### Setup -->
+  <!-- Import the amp-user-notification component in the header. -->
+  <script async custom-element="amp-user-notification" src="https://cdn.ampproject.org/v0/amp-user-notification-0.1.js"></script>
+  <!--
+    Specify any additional css, default css class will put the amp-user-notification element
+    in the left bottom corner.
```

Comment:
replace 'default css class will put the amp-user-notification element in the left bottom corner' with 'by default, the `amp-user-notification` is positioned in the left bottom corner.